doc/nrf/known_issues.rst

```diff
@@ -374,7 +374,7 @@ NCSDK-8223: GATT requests might deadlock RX thread
   This causes a deadlock because only the RX thread releases the TX buffers for the GATT requests.
   The deadlock is resolved by a 30 second time-out, but the ATT bearer cannot transmit without reconnecting.
 
-  **Workaround:** Set :option:`CONFIG_BT_L2CAP_TX_BUF_COUNT` >= :option:`CONFIG_BT_ATT_TX_MAX` + 2.
+  **Workaround:** Set :option:`CONFIG_BT_L2CAP_TX_BUF_COUNT` >= ``CONFIG_BT_ATT_TX_MAX`` + 2.
```

Comment:
@joerchan Does this workaround still make sense if the option does not exist anymore?

Reply:
The workaround makes sense for older releases. We are pulling in the fix now.